Composer/packages/types/src/indexers.ts

```diff
@@ -150,6 +150,11 @@ export type LgParsed = {
   templates: LgTemplate[];
 };
 
+export type ImportExpression = {
```

Comment:
Avoid the work expression here as it has other meaning in bot framework.  How about LuLgImport, LanguageImport, or even MarkdownImport?